cmd/revision.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/databus23/helm-diff/diff"
+	"github.com/databus23/helm-diff/manifest"
+	"github.com/mgutz/ansi"
+	"github.com/spf13/cobra"
+	"k8s.io/helm/pkg/helm"
+)
+
+const revisionCmdLongUsage = `
+This command compares the manifests details of a named release.
+
+It can be used to compare the manifests of 
+ 
+ - lastest REVISION with specified REVISION
+	$ helm diff revision [flags] [RELEASE] [REVISION]
+
+ - REVISION1 with REVISION2
+	$ helm diff revision [flags] [RELEASE] [REVISION1] [REVISION2]
+`
+
+func revisionCmd() *cobra.Command {
+	diff := diffCmd{}
```

Comment:
I would prefer if this could use its own struct and not piggy back on the exiting struct. The general Idea would be that every command has its own struct, to store state und functions.

Reply:
each subcommand is given a client, as off now both has resemblance, but it could change in feature.
```go
# cmd/revision.go
type revision struct {
	release         string
	client          helm.Interface
	suppressedKinds []string
	revisions       []string
}
```
```go
# cmd/rollback.go
type rollback struct {
	release         string
	client          helm.Interface
	suppressedKinds []string
	revisions       []string
}
```